public/style.css

```diff
@@ -15,3 +28,57 @@
 ul {
   list-style: none;
 }
+
+#list-of-rules{
+  list-style:square;
+  text-align: left;
+  margin-left: 20px;
+  margin-right: 40px;
+  background-color: #DBE2EF;
```

Comment:
This can be deleted because same background colour is already being applied by parent element styling on line 66.